Module3_3/Module3_3/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Module3_3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Введите конец диапазона чисел Фибоначчи от 1 до : ");
+            int n = int.Parse(Console.ReadLine());
+            int u, u1 = 0, u2 = 1;
+            Console.WriteLine(u1);
```

Comment:
you can invoke it once: 
Console.WriteLine("{0}{1}{2}",a, Environment.NewLine, b); or Console.WriteLine(u1 + "\t" + u2);
always try to reduce number of methods invocations